### DISCUSSION TITLE
Restrict File Types When Creating Field

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#12437] [Archive]
Hi,

Is it possible to limit that file types when you create a Files field? I tried creating a condition to make sure it has a width or that the mimetype should contain "image" but it didn't do anything.

Given that you are allowed to choose the Files root, a workaround exists if you could limit the types of files in a particular directory. 

thanks
  Matt

**Comment by rijkvanzanten** [comment]
There's currently no configuration to do that on the field level
